### DISCUSSION TITLE
USB drive install

### DISCUSSION THREAD

**Discussion by Diamondog** [keepassxreboot/keepassxc#9005] [Q&A] [answered]
Is there a way to install Keepassxc onto a USB drive and have it work on Linux and Windows? 
I want to be able to use it on both OS with one USB. How can this be done and is there also a way to do this 
with my Android device? via NFC or something?

**Comment by droidmonkey** [comment, marked as answer]
You'll need both the windows portable and the Linux appimage in that USB drive

**Reply by Diamondog** [reply]
Can you give direction on how I would do this?

**Reply by droidmonkey** [reply]
Just download both, unzip the portable to the drive and copy the appimage to the drive. On Linux make the appimage executable. Then just run the apps.

**Reply by Diamondog** [reply]
Why are there no updates to the software at this time? is anyone actively working on this ?

**Reply by Diamondog** [reply]
Never mind that-- I just noticed you were working on it :)

**Reply by Diamondog** [reply]
How can I chat with you LIVE do you have telegram or some way to text?
